src/server/config.rs

```diff
@@ -232,7 +236,7 @@ impl Default for Config {
             ),
             end_point_max_concurrency: cmp::max(cpu_num as usize, MIN_ENDPOINT_MAX_CONCURRENCY),
             snap_max_write_bytes_per_sec: ReadableSize(DEFAULT_SNAP_MAX_BYTES_PER_SEC),
-            snap_max_total_size: ReadableSize(0),
+            snap_max_total_size: ReadableSize(DEFAULT_SNAP_MAX_TOTAL_SIZE),
```

Comment:
Can it be adjusted according to the region size? For example, if we want to allow 8 concurrent snapshots but each snapshot can take 10 GiB, then the maximum size should be 80GiB.

Reply:
mm, but for smaller region, we probably want to have bigger concurrent snapshots to speed up the process. 
I think use a size is a more clear mechanism and easier to understand.